index.js

```diff
@@ -34,12 +34,14 @@ async function run() {
     const json = JSON.parse(data);
 
     const coverage = `<!--json:${JSON.stringify(originMeta)}-->
-|${inputs.title}| %                           | values                                                              |
```

Comment:
Dear reviewer, this is the fundamental code change, to restyle the coverage results table.